### DISCUSSION TITLE
Monterey

### DISCUSSION THREAD

**Discussion by compulsivebuyer** [Ascoware/get-iplayer-automator#362] [Announcements]
Evening all, if I upgrade from Big Sur to Monterey will GIA still work?

**Comment by skovatch** [comment]
Yes, GiA works fine on Monterey. The last release or two was actually built on early versions of Monterey. I will note, however, that macOS may complain about the app needing to be updated if you download ITV shows. This is because python 2.7 is still the default version, and I can't assume python 3.x is available on older versions of macOS.

**Reply by compulsivebuyer** [reply]
Many thanks Scott
